docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -144,21 +112,50 @@ at the root of `project.json`:
 }
 ```
 
-Finally, you need to set the framework node to use
+The template also sets the framework node to use
 `netcoreapp1.0`, and include the required imports to
-get xUnit.net to work with RC2:
+get xUnit.net to work with .NET Core RTM:
 
 ```json
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
```

Comment:
I don't think imports aren't needed anymore.  Haven't had to write a test using them since RTM released.